Demo/Demo/Models/CreateOrderParams.swift

```diff
@@ -3,7 +3,7 @@ struct CreateOrderParams: Encodable {
     let applicationContext: ApplicationContext?
     let intent: String
     var purchaseUnits: [PurchaseUnit]?
-    var paymentSource: VaultCardPaymentSource?
+    var paymentSource: VaultPaymentSource?
```

Comment:
Just to clarify my understanding - a merchant only needs to set the `paymentSource` in their create order call if they want to vault, correct?

How do we document this to a merchant?

Reply:
Yes, JS SDK has documentation on this on their developer site. We should do something similar.